index.html

```diff
@@ -17,10 +17,52 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1 class="heading">Code Journal</h1>
+          </div>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <form class="column-full">
+        <div class="row">
+          <div class="column-half">
+            <h2 class="newEntry">New Entry</h2>
+
+            <div data-view="entry-form">
```

Comment:
This needs to be the first element after `<main>`. All of your content needs to go inside of here